src/internal/internal_tzcopy.cc

```diff
@@ -298,8 +298,11 @@ void copy(internal::TargetType<Target::Devices>,
                             B_diag_tiles.insert( { i, j } );
                         }
                         else {
+                            auto tileDevice =
```

Comment:
Should be `tile_device`, right? I would probably just say `int` instead of `auto`.

Reply:
My bad. Fixed.